xedocs/schemas/corrections/implementations/position_reconstruction.py

```diff
@@ -24,7 +24,7 @@ class PosRecModel(BaseResourceReference):
     _ALIAS = "posrec_models"
     fmt = "binary"
 
-    kind: Literal["cnn", "gcn", "mlp", "s1_cnn", "flow"] = rframe.Index()
+    kind: Literal["cnn", "gcn", "mlp", "s1_cnn", "flow", "cnf"] = rframe.Index()
```

Comment:
I believe `"flow"` is not needed. `"cnf"` is not needed now, but it will be needed after https://github.com/XENONnT/straxen/issues/1506 is solved.